src/libs/recaptcha/ReCaptcha.web.tsx

```diff
@@ -56,37 +56,33 @@ export function ReCaptcha(props: Props) {
       const { grecaptcha } = window
       const reCaptchaContainer = reCaptchaContainerRef.current
       const isReCaptchaRendered = reCaptchaContainer?.hasChildNodes()
-      if (
-        reCaptchaContainer &&
-        grecaptcha &&
-        grecaptcha.ready &&
-        grecaptcha.render &&
-        grecaptcha.execute
-      ) {
-        if (!isReCaptchaRendered) {
-          grecaptcha.ready(() => {
-            if (grecaptcha.render) {
-              reCaptchaWidgetRef.current = grecaptcha.render(reCaptchaContainer, {
-                sitekey: env.SITE_KEY,
-                callback: onSuccess,
-                'expired-callback': props.onExpire,
-                'error-callback': onRecaptchaErrorCallback,
-                size: 'invisible',
-                theme: 'light',
-              })
-            }
-          })
-        }
-        if (isReCaptchaRendered) {
-          try {
-            grecaptcha.execute(reCaptchaWidgetRef.current)
-            clearInterval(intervalId)
-          } catch (error) {
-            if (error instanceof Error)
-              props.onError(
-                ReCaptchaInternalError.UnknownError,
-                'reCAPTCHA error: ' + error.message
-              )
+
+      if (!grecaptcha?.ready || !grecaptcha.render || !grecaptcha.execute) {
+        return
+      }
+
+      if (!isReCaptchaRendered && reCaptchaContainer) {
+        grecaptcha.ready(() => {
+          if (grecaptcha.render) {
```

Comment:
Pas besoin de faire ce check vu que tu le fais avant

Reply:
Je pense que c'est parce que je suis dans le scope de `grecaptcha.ready` que si je le met pas, j'ai "Cannot invoke an object which is possibly 'undefined'.ts(2722)" sur `grecaptcha.render`